dev/com.ibm.ws.kernel.boot.core/src/io/openliberty/checkpoint/spi/CheckpointPhase.java

```diff
@@ -124,7 +149,7 @@ final public boolean restored() {
      *
      * @param hook the hook to be used to prepare and restore
      *                 a checkpoint process.
-     * @throws IllegalStateException if the phase is in progress.
+     * @throws IllegalStateException if the phase is not in progress.
      * @see CheckpointHook#MULTI_THREADED_HOOK
```

Comment:
This javadoc also needs to be updated to say what the return value means (same for `addMultiThreadedHook()`)

Reply:
Thanks for pointing that out.  Here is the update I made which is applied to both methods:

```java
    /**
     * Adds a {@code CheckpointHook} for this checkpoint phase.
     * The hook will be run to {@link CheckpointHook#prepare() prepare}
     * and {@link CheckpointHook#restore() restore} a checkpoint
     * process while the JVM is in multi-threaded mode. If the
     * checkpoint process is already {@link #restored() restored}
     * then the hook is not added and {@code false} is returned,
     * indicating that the hook will not be called.
     * A return value of {@code true} indicates that the hook will
     * be called for both {@link CheckpointHook#prepare() prepare}
     * and {@link CheckpointHook#restore() restore}.
     *
     * @param hook the hook to be used to prepare and restore
     *                 a checkpoint process.
     * @throws IllegalStateException if the phase is not in progress.
     * @return true if the hook is successfully added; otherwise false
     *         is returned.
     * @see CheckpointHook#MULTI_THREADED_HOOK
     */
```